pkg/build/graph/edges.go

```diff
@@ -55,25 +58,47 @@ func AddInputOutputEdges(g osgraph.MutableUniqueGraph, node *buildgraph.BuildCon
 		g.AddEdge(in, node, BuildInputEdgeKind)
 	}
 
-	from := buildutil.GetImageStreamForStrategy(node.BuildConfig.Spec.Strategy)
-	if from != nil {
-		switch from.Kind {
-		case "DockerImage":
-			if ref, err := imageapi.ParseDockerImageReference(from.Name); err == nil {
-				tag := ref.Tag
-				ref.Tag = ""
-				in := imagegraph.EnsureDockerRepositoryNode(g, ref.String(), tag)
-				g.AddEdge(in, node, BuildInputImageEdgeKind)
+	inputImages := []*kapi.ObjectReference{}
+	triggerImages := []*kapi.ObjectReference{}
+	inputImage := buildutil.GetImageStreamForStrategy(node.BuildConfig.Spec.Strategy)
```

Comment:
yes, and it's used if a trigger is found with no From reference here: https://github.com/openshift/origin/pull/3748/files#diff-66216bd1fb27dad14075a75c1f66cdcaR73